src/Support/Support.Autorest/README.md

```diff
@@ -60,39 +57,83 @@ directive:
     set:
       format-table:
         properties:
-          # - Id
           - DisplayName
           - Name
-          - SecondaryConsentEnabled # Do we need this? 
-          # - Type
+          - SecondaryConsentEnabled
```

Comment:
why do we have this here?

Reply:
this was listed in the swagger as a problem classification property-do we not need this?

Reply:
did not see the model-name for problem classification property. Does that need to be added in that case?